cmd/ocm-backplane/cloud/credentials.go

```diff
@@ -15,6 +15,10 @@ import (
 	"github.com/openshift/backplane-cli/pkg/utils"
 )
 
+var GetBackplaneClusterFromConfig = utils.DefaultClusterUtils.GetBackplaneClusterFromConfig
```

Comment:
I guess this 1-1 map completely gets rid of currently since tests are not dependent on these variables.

Reply:
We can clean this up in another PR since Hector is OOO